Engineering outfit resprites

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
Resprites to the Engineering Department!
Adds the Ce's beanie, engineering windbreaker & adds a senior atmos jumpsuit/skirt
Adds hi-visibility to the engineering wintercoats because visibility isn't a good substitute for protection against the elements!
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Engineers need a reason to not wear their suit when the station complains things aren't being fixed.
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="384" height="336" alt="Screenshot 2025-12-10 155624" src="https://github.com/user-attachments/assets/f3289790-e182-43b6-842e-ecf223cb7a9a" />
<img width="328" height="218" alt="Screenshot 2025-12-10 161935" src="https://github.com/user-attachments/assets/4b31751b-ea58-425d-a56c-0e6fac743ba4" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Upstream Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html) as well as the [Moffstation Contributing Guidelines](https://github.com/moff-station/moff-station-14/blob/master/CONTRIBUTING.md).
- [X] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: ce's beanie, engineering windbreaker & senior atmos outfits
- tweak: ce's outfits, engineer & atmos outfits

Davyei committed:
engineering sprites

Davyei committed:
Engineer Clothes!

Centronias commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -1010,6 +1011,7 @@
   #- StationEngineerHardhatYellow Moffstation - (Work Gear for Atmos) - Yellow and blue hardhats are restricted to atmos to keep the two roles visually distinct.
   - StationEngineerHardhatOrange
   - StationEngineerHardhatRed
+  - StationEngineerOrangeCap
```
```suggestion
  - StationEngineerOrangeCap # Moffstation - Engineering Outfits
```

Centronias commented on Resources/Prototypes/_Moffstation/Entities/Clothing/OuterClothing/vests.yml:
```diff
@@ -1,24 +1,39 @@
   id: ClothingOuterVestHazardBlue
   name: hi-viz vest
-  description: A high-visibility vest used in work zones. This one is atmos-themed blue. How prestigious.
+  description: A high-visibility vest used in work zones.
```
Taking away my prestige 😭

Centronias commented on Resources/Prototypes/_Moffstation/Entities/Clothing/OuterClothing/vests.yml:
```diff
@@ -1,24 +1,39 @@
 ﻿#Hazard vest
 - type: entity
-  parent: [ ClothingOuterVestHazard ]
+  parent: [ ClothingOuterStorageBase, AllowSuitStorageClothing ]
```
Why not inherit?

Davyei replied:
it didn't work when I 'was' trying to do it that way. may work with an inherit but let me see rq

Centronias reviewed (changes requested):
I only looked at the first RSI, I'm trusting the linter for the rest.

Davyei committed:
lathe additions

Davyei committed:
Update Resources/Prototypes/Loadouts/loadout_groups.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/Loadouts/loadout_groups.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Centronias Requests

- Meta.jsons which I can update the copyright on have been updated

- Hi-vis vest given back it's prestige & inherits without breaking

Davyei committed:
Merge branch 'engineering-outfit-resprites' of https://github.com/Davyei/moff-station-14 into engineering-outfit-resprites

Davyei committed:
Big idiot fix

Centronias committed:
Update Resources/Textures/_Moffstation/Clothing/OuterClothing/Vests/hazard.rsi/meta.json

Signed-off-by: Centronias <charlie.t.santos@gmail.com>
